symplyphysics/core/symbols/symbols.py

```diff
@@ -54,7 +55,19 @@ def display_latex(self) -> str:
         return self._display_latex
 
     def _sympystr(self, p: Printer) -> str:
-        return p.doprint(self.display_name)
+        return p.doprint(self.display_name)    
+
+
+class DimensionSymbolNew(HasDimension, HasDisplay):
```

Comment:
This is not needed since you've extracted all logic somewhere. Now it's a wrapper that doesn't do anything.